cmd/snap/cmd_advise.go

```diff
@@ -205,11 +210,50 @@ func adviseViaAptHook() error {
 	return nil
 }
 
+type Snap struct {
+	Snap    string
+	Version string
+	Command string
+}
+
+func dumpDbHook() error {
+	commands, err := advisor.DumpCommands()
+	if err != nil {
+		return err
+	}
+
+	commands_processed := make([]string, 0)
```

Comment:
`var processed []string`

Or commandsProcessed if you're keen on it. The underline is atypical for Go.

The _make([]foo, 0)_ is also almost always unwanted. The behavior of Go with a nil slice is very similar to that of a slice with no items (len works, etc).